File: apps/my-service-admin/src/api/customer/Customer.ts
import { Order } from "../order/Order";

export type Customer = {
  id: string;
  createdAt: Date;
  updatedAt: Date;
  firstName: string | null;
  lastName: string | null;
  email: string | null;
  phoneNumber: string | null;
  orders?: Array<Order>;
  customerMetadata: string | null;
};

File: apps/haim/src/APIs/Customer/Dtos/Customer.cs
namespace Haim.APIs.Dtos;

public class Customer
{
    public string Id { get; set; }

    public DateTime CreatedAt { get; set; }

    public DateTime UpdatedAt { get; set; }

    public string? Name { get; set; }

    public string? LastName { get; set; }

    public List<string>? Orders { get; set; }
}
